### PR TITLE
add support for customizable flash messages

### PR DESCRIPTION
**Pull request by JoeWoodward**
add option to hide close toggle, or set custom text
add option to insert custom style
add more info to README

### CODE REVIEW

File: lib/foundation_rails_helper/flash_helper.rb [hunk 1]
```diff
@@ -7,38 +7,43 @@ module FlashHelper
...
-      :success   => :success,
-      :error     => :alert,
-      :warning   => :warning
+      :alert     => 'alert',
+      :notice    => 'success',
+      :info      => 'info',
+      :secondary => 'secondary',
+      :success   => 'success',
+      :error     => 'alert',
+      :warning   => 'warning'
```
Comment (dsandstrom) [inline lib/foundation_rails_helper/flash_helper.rb]: Why were these changed to strings?

File: lib/foundation_rails_helper/flash_helper.rb [hunk 2]
```diff
@@ -7,38 +7,43 @@ module FlashHelper
...
       key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
-      key_matching.default = :standard
+
+      dismiss_toggle = opts[:dismiss_toggle]
+      alert_style = opts[:style]
 
       capture do
         flash.each do |key, value|
           next if FoundationRailsHelper.configuration.ignored_flash_keys.include? key.to_sym
           alert_class = key_matching[key.to_sym]
```
Comment (JoeWoodward) [inline lib/foundation_rails_helper/flash_helper.rb]: @dsandstrom this is the only place the hash is used. I can't see any reason that they should be symbols

Comment (dgmstuart) [inline lib/foundation_rails_helper/flash_helper.rb]: @JoeWoodward Happy to change it if there's a good reason, but I'd argue that existing code shouldn't be changed _without_ a good reason :wink:

File: lib/foundation_rails_helper/flash_helper.rb [hunk 3]
```diff
@@ -7,38 +7,43 @@ module FlashHelper
...
+      :notice    => 'success',
+      :info      => 'info',
+      :secondary => 'secondary',
+      :success   => 'success',
+      :error     => 'alert',
+      :warning   => 'warning'
     }
-    def display_flash_messages(key_matching = {})
+
+    def display_flash_messages(opts={})
```
Comment (dgmstuart) [inline lib/foundation_rails_helper/flash_helper.rb]: @JoeWoodward Wait, this is a breaking change isn't it? Existing user code which calls `display_flash_messages` will break

**Comment by dgmstuart** [comment]
@JoeWoodward Can you try rebasing your branch please: see if we can get it building

File: lib/foundation_rails_helper/flash_helper.rb (hunk 3 above)
Comment (dsandstrom) [inline lib/foundation_rails_helper/flash_helper.rb]: Good point.  Use `display_flash_messages(key_matching = {}, opts = {})`.  The options can be combined in version 2.0.

**Comment by dsandstrom** [comment]
Closing because of lack of response.
